### PR TITLE
Implement Developer Hub API on API Information Page

### PR DESCRIPTION
**Pull request by notatiyyah**
Implement Developer Hub API on API Information Page. The system uses the logged-in user's hackney token from their cookies for authorisation.

- [x] Fetch additional information about APIs from the Developer Hub API
- [x] Add error handling if either API fails
- [x] Test added functionality

Preview changes here: https://developer-api-development.hackney.gov.uk/

### CODE REVIEW

File: src/pages/apiInformation/apiInformation.page.jsx [hunk 1]
```diff
@@ -11,109 +14,151 @@ import Breadcrumbs from "../../components/breadcrumbs/breadcrumbs.component.jsx"
...
-                            </div>
-                        </>
-                )
-                : 
-                <h3>Loading..</h3>}
+                <div className="sidePanel">
+                    <Breadcrumbs />
+                    <h1>
+                        {!apiStatus.error && (apiData.apiName || <Skeleton/>)}
+                        {(apiStatus.error && (swaggerStatus.isLoaded ? swaggerData.info.title : <Skeleton/>))}
```
Comment (1JW1) [inline src/pages/apiInformation/apiInformation.page.jsx:143]: The loading skeleton is a lot easier to use than I thought, nice library find

Comment (notatiyyah) [reply src/pages/apiInformation/apiInformation.page.jsx:143]: @1JW1 It's really neat! And a great way to load stuff from multiple APIs with different latency, etc.

**Review by humulla — COMMENTED** [review]
Looks good to me! But, you need to resolve the merge conflicts first

**Comment by humulla** [comment]
Seeing as the issue with the API in dev has now been resolved, can we please update the dev frontend to call the development API

**Comment by notatiyyah** [comment]
> Seeing as the issue with the API in dev has now been resolved, can we please update the dev frontend to call the development API

@humulla already done, just needed to update the URL in parameter store :)

**Review by humulla — APPROVED** [review]

**Pull request merged**